packages/core/src/components/buttons/buttons.module.scss

```diff
@@ -10,53 +10,50 @@
 
 .appearance-primary {
 	color: white;
-	background: $colors-primary-a2 !important;
 	border: none;
 	border-radius: 8px;
 
 	&:hover {
-		background: $colors-accents-2 !important;
+		background: $colors-accents-2;
 	}
 
 	&:focus {
-		background: $colors-warning-1!important;
+		background: $colors-warning-1;
 		color: $colors-neutral-8;
 		box-shadow: 0px 2px 0px $colors-neutral-8;
+		outline: none;
 	}
-}
-
-.appearance-primary:disabled {
-	cursor: not-allowed !important;
-	background: $colors-primary-a1;
 
-	&:hover {
-		background: $colors-neutral-6 !important;
+	&:disabled {
+		cursor: not-allowed !important;
+		background: $colors-neutral-6;
+	
+		&:hover {
+			background: $colors-neutral-6 !important;
+		}
 	}
 }
 
 .appearance-secondary {
 	color: white;
-	background: $colors-primary-3;
+	background: $colors-primary-3!important;
 	border: none;
 	border-radius: 8px;
 
-	&:hover {
-		background: $colors-neutral-a1 !important;
-	}
-
 	&:focus {
 		background: $colors-warning-1!important;
-		color: $colors-neutral-8;
+		color: white;
```

Comment:
Could instances of `white` be changed to the white stored in `variables.scss` to ensure we are using the same white throughout. Its unlikely to change but if it does in the future the color will stay consistent throughout.

Reply:
Well spotted. White was the wrong colour here anyway!